cub/cub/detail/rfa.cuh

```diff
@@ -0,0 +1,950 @@
+/******************************************************************************
+ * Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Time flies fast
```suggestion
 * Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Could we use a shorten license with just the tags? @bernhardmgruber

Reply:
I have conflicting information regarding this and cannot give any guidance here :/